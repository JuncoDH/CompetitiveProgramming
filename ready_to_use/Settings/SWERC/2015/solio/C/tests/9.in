20
1
1
2
1 1
3
3 1 1
4
1 5 3 1
5
3 1 1 5 9
6
1 9 5 15 3 1
7
1 9 1 3 15 25 5
8
1 3 5 1 15 25 41 9
9
15 5 67 3 25 1 41 1 9
10
9 1 25 1 109 41 15 3 5 67
11
41 15 25 177 1 109 67 1 5 9 3
12
67 109 5 3 41 1 25 287 9 1 15 177
13
3 109 9 287 1 5 465 1 41 15 67 177 25
14
41 15 465 177 25 9 1 67 753 287 109 1 3 5
15
753 1 177 67 15 1 5 41 25 1219 109 465 9 287 3
16
287 753 15 177 5 109 1219 25 465 41 3 67 1 9 1 1973
17
67 177 9 1973 1219 25 287 465 5 15 3 3193 109 1 1 41 753
18
753 5 109 41 5167 15 3 465 67 1 177 1973 9 3193 1219 1 287 25
19
15 177 465 1219 1973 8361 41 109 3 25 67 287 753 3193 5167 1 5 1 9
20
753 67 1 5167 465 177 3193 13529 5 1219 1973 287 41 3 9 15 8361 109 1 25
