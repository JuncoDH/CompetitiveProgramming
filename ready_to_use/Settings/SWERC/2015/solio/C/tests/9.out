0
2
7
17
36
70
129
229
396
672
1125
1865
3070
5028
8205
13355
21698
35208
57079
92479
